server/src-lib/Hasura/App.hs

```diff
@@ -228,7 +229,7 @@ initialiseCtx env (HGEOptionsG rci metadataDbUrl hgeCmd) = do
     migrateCatalogSchema logger pgLogger env connParams rci metadataDbUrl
 
   schemaCacheE <- runExceptT
-    $ peelMetadataRun (RunCtx adminUserInfo httpManager sqlGenCtx) metadata
+    $ peelMetadataRun (RunCtx adminUserInfo httpManager sqlGenCtx {- TODO: -} defaultResolveCustomSource) metadata
```

Comment:
Here, we initialize and build the `RebuildableSchemaCache` for the OSS server. No other customization exist (via type classes). We can use default here for OSS.

Reply:
Isn't this also where we build the cache for Pro then? So wouldn't we need to add the type class here?

Reply:
Or do you mean it's just the connection used to read the Postgres metadata? I guess what I'm asking is - will the `ResolvedSource`s be stored in the schema cache?

Reply:
> Isn't this also where we build the cache for Pro then?
I'm not sure about this. If yes, then we should add the type class here too.